op-chain-ops/genesis/config.go

```diff
@@ -443,6 +445,9 @@ func (d *DeployConfig) Check() error {
 		if d.DAResolveWindow == 0 {
 			return fmt.Errorf("%w: DAResolveWindow cannot be 0 when using plasma mode", ErrInvalidDeployConfig)
 		}
+		if !(d.DACommitmentType == 0 || d.DACommitmentType == 1) {
```

Comment:
Seems like we should have some constants defined for these values instead of using 0 and 1 directly?

Reply:
And actually in the json it would be nice if this wasn't just a meaningless number. Typically would use a string in json config for things like this.

Reply:
This is the bad habit of using enum values in config. I switched over to a string.